### DISCUSSION TITLE
Do starred entries get deleted during database cleanups?

### DISCUSSION THREAD

**Discussion by DonkeeeyKong** [Athou/commafeed#1606] [Q&A] [answered]
Hello, I recently set up a selfhosted instance of commafeed (very nice work! Thanks to everybody involved! :) ) and now I am wondering if `COMMAFEED_DATABASE_CLEANUP_ENTRIES_MAX_AGE` and `COMMAFEED_DATABASE_CLEANUP_MAX_FEED_CAPACITY` apply to all entries without exceptions or if starred entries and entries with a tag are kept regardless of these settings. 
Thanks in advance!

**Comment by Athou** [comment, marked as answer]
The fact that an entry is starred or not has currently no impact on cleanup tasks. There is an issue to improve this: https://github.com/Athou/commafeed/issues/1581

**Reply by DonkeeeyKong** [reply]
Thank you! 

After thinking a bit more about it: Using something like Linkding is probably the better solution for long term archiving of bookmarks anyway. :)
